src/src/Helpers/Modbus_device.cpp

```diff
@@ -60,8 +67,7 @@ bool ModbusDEVICE_struct::init(uint8_t                 slaveAddress,
     log += _deviceID;
     addLogMove(LOG_LEVEL_INFO, log);
```

Comment:
More `strformat()` logging, so no new String has to be created and concatenated multiple times, causing re-allocations